sunpy/map/mapbase.py

```diff
@@ -1319,13 +1319,20 @@ def rotation_matrix(self):
         It general it does not have to be a pure rotation matrix, and can encode
         other transformations e.g., skews for non-orthgonal coordinate systems.
         """
-        if 'PC1_1' in self.meta:
-            return np.array([[self.meta['PC1_1'], self.meta['PC1_2']],
-                             [self.meta['PC2_1'], self.meta['PC2_2']]])
-
-        elif 'CD1_1' in self.meta:
-            cd = np.array([[self.meta['CD1_1'], self.meta['CD1_2']],
-                           [self.meta['CD2_1'], self.meta['CD2_2']]])
+        if any(key in self.meta for key in ['PC1_1', 'PC1_2', 'PC2_1', 'PC2_2']):
```

Comment:
```suggestion
        if {'pc1_1', 'pc1_2', 'pc2_1', 'pc2_2']} & self.meta.keys():
```
In general, I'd like to see more usage of set logic.  That said, in this case, it's probably not any more readable, and it has the further annoyance of being case-sensitive since it can't make use of the overloaded `in`.  Feel free to ignore this suggestion.